app/views/templates/admin_index.html.erb

```diff
@@ -83,12 +83,12 @@
     <a href="<%= admin_new_template_path %>" class="btn btn-primary" role="button">
       <%= _('Create a template') %>
     </a>
+    <% if !current_user.org.organisation? %>
```

Comment:
All orgs should be able to see the default template in this list (unless they are the owner), so other organisations, funders and institutions should be able to customize the default.
@sjDCC can clarify if necessary